miqa/urls.py

```diff
@@ -48,6 +51,11 @@
     path('api/docs/swagger/', schema_view.with_ui('swagger'), name='docs-swagger'),
 ]
 
+if os.environ['DJANGO_CONFIGURATION'] == 'DevelopmentConfiguration':
```

Comment:
Rather than check the configuration, let's make an explicit setting called `HOMEPAGE_REDIRECT_URL`. We should add it to the config mixin as an env var and set it to `http://localhost:8081` in DevelopmentConfiguration.

(@dchiquito  is that right?)

Reply:
we could put something like `HOMEPAGE = HomePageView.as_view()` in MiqaMixin and `HOMEPAGE = RedirectView.as_view(url='http://localhost:8081')` in DevelopmentConfiguration?

Reply:
Sure, that sounds good. 👍

Reply:
I did run into something for this. I noticed that this works fine for the DevelopmentConfiguration, where all we need is RedirectView imported. However, for all other configurations inheriting from MiqaMixin, we need to import HomePageView from `miqa.core.rest.home`. When such an import is made anywhere in settings.py, we get this error:
`django.core.exceptions.AppRegistryNotReady: Couldn't setup configuration 'miqa.settings.DevelopmentConfiguration':  Apps aren't loaded yet.`

I couldn't find a workaround that would allow us to do lazy-loading on this import (even though this import doesn't even reference any models). Thankfully, the functionality of HomePageView is really simple, so what we _could_ do is use TemplateView directly: `return path('', TemplateView.as_view(template_name='index.html'), name='home')`. In this case, we wouldn't even need the file miqa/core/rest/home.py, though deleting it would lose the comment contents...

What do you think, @zachmullen?

Reply:
@dchiquito and I were able to figure out a new solution where the settings configurations can either specify a string URL or `None` for a value called `HOMEPAGE_REDIRECT_URL`. home.py returns the original `TemplateView` if this value is `None` or a `RedirectView` to the URL otherwise. We figured that this solution is both DRY and Configuration-flexible.